templates/layout.html

```diff
@@ -94,6 +94,9 @@
                 <li {% block current_page_class_faq %}{% endblock %}>
                     <a href="{{ pathto('faq') }}">FAQ</a>
                 </li>
+                <li {% block current_page_class_gdpr %}{% endblock %}>
+                  <a href="{{ pathto('gdpr') }}">GDPR</a>
```

Comment:
GDPR je sice název směrnice, ale asi tam mělo být něco jako _Ochrana osobních údajů_.

Ale to by se asi nahoře nevešlo, už tohle je tak tak.

![image](https://user-images.githubusercontent.com/334494/118483144-f9518e80-b715-11eb-8414-5fd29aaafe43.png)

Co to dát jinam?

Reply:
Třeba na patičku stránky?